src/services/cherry-picker.ts

```diff
@@ -392,10 +400,11 @@ export class CherryPicker {
           }
         }
       }
-
+      // console.log(`${weightFactor}: attempts: ${sortedLog.attempts} success: ${Math.round(sortedLog.successRate * 100)}% median: ${sortedLog.medianSuccessLatency}s weighted: ${sortedLog.weightedSuccessLatency} ${sortedLog.id}`)
```

Comment:
is this meant to stay?

Reply:
Yea it made checking the results easy. it's commented out.

Reply:
if you also want it to be shown on prod, will need to change it to `logger.log` unless you only want it for local testing

Reply:
it's commented out, it is just for me to use locally later. i'll delete it in the code.